### DISCUSSION TITLE
Prometheus Metric - Time since last backup

### DISCUSSION THREAD

**Discussion by itay-grudev** [cloudnative-pg/cloudnative-pg#1713] [General]
I would like to implement monitoring and alarms based on the time since the last successful backup of my database cluster. If the time exceeds a threshold - meaning a backup wasn't finished successfully I would like oncall personnel to look into it.

Can you propose a method for implementing such a metric?

**Comment by gbartolini** [comment]
Hi @itay-grudev,

We have implemented already such metrics, which will be available in the next release (1.19.1). See this commit: https://github.com/cloudnative-pg/cloudnative-pg/commit/8ab810ad08227d7049c19da59c6209dd81b0c52e

Please let me know if they are enough.

Cheers,
Gabriele

**Reply by itay-grudev** [reply]
Perfect. Looking forward to it.
